internal/generate/openapi/openapi_test.go

```diff
@@ -0,0 +1,33 @@
+package openapi
+
+import (
+	"bytes"
+	"io/fs"
+	"os"
+	"testing"
+
+	"cuelang.org/go/cue"
+	"github.com/georgemac/truce"
```

Comment:
Looks like there are some stray imports from the old way.

Reply:
gross, good spot